clients/cobol-lsp-vscode-extension/src/extension.ts

```diff
@@ -76,6 +78,18 @@ export async function activate(context: vscode.ExtensionContext) {
         editDatasetPaths(pathsService);
     }));
 
+    //to be deleted after implementation
+    context.subscriptions.push
+    (vscode.commands.registerCommand("broadcom-cobol-lsp.cpy-manager.resolve-git-copybooks", () => {
+        prioritizer.checkCopybooksPresentLocal(["DEMO1", "DEMO2"],
+             vscode.workspace.getConfiguration(SETTINGS_SECTION).get(PATHS_LOCAL_KEY));
+    }));
```

Comment:
is not clear which is the value of this command, with two hardcoded copybook names

Reply:
to trigger the method

Reply:
But it could be engaged from ```fetchCopybook``` cmd right?